### PR TITLE
app-admin/doas: Bump version to 6.6.1

### PR DESCRIPTION
**Pull request by realaltffour**
Bumped version to 6.6.1 to support new features.

Package-Manager: Portage-2.3.84, Repoman-2.3.20

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
You need a proper commit message with signoff

File: app-admin/doas/doas-6.6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline app-admin/doas/doas-6.6.1.ebuild]: Use the latest available EAPI, which is 7.

File: app-admin/doas/doas-6.6.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~arm"
+IUSE="pam"
+
+RDEPEND="pam? ( sys-libs/pam )"
+DEPEND="${RDEPEND}
+	virtual/yacc"
+
+src_prepare()
+{
```
Comment (juippis) [inline app-admin/doas/doas-6.6.1.ebuild]: Stick to Gentoo's common syntax `src_prepare() {`

File: app-admin/doas/doas-6.6.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	default
+	sed -i 's/-Werror //' Makefile || die
+}
+
+src_configure()
+{
```
Comment (juippis) [inline app-admin/doas/doas-6.6.1.ebuild]: Same here.

File: app-admin/doas/doas-6.6.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	tc-export CC AR
+	./configure \
```
Comment (juippis) [inline app-admin/doas/doas-6.6.1.ebuild]: Won't `econf` work? It should also provide some defaults you define below.

Comment (realaltffour) [reply app-admin/doas/doas-6.6.1.ebuild]: No, I have tried it with ```econf``` and it failed with: No ```info-dir``` command available. The ```configure``` file is not an  ```autconf``` script.

**Comment by juippis** [comment]
5cd52ba5291c71e801b5b49eb665efdcf95183a5

**Pull request closed without merging**